src/test/ui/hygiene/fields.stderr

```diff
@@ -1,35 +1,35 @@
-error: type `foo::S` is private
+error: struct `foo::S` is private
   --> $DIR/fields.rs:15:17
    |
-LL |         let s = S { x: 0 }; //~ ERROR type `foo::S` is private
-   |                 ^^^^^^^^^^
+LL |         let s = S { x: 0 };
+   |                 ^^^^^^^^^^ private
```

Comment:
this note does not give the user any new info, I think it should just be left out (like it was originally)